cmd/frakti/frakti.go

```diff
@@ -58,6 +58,7 @@ var (
 		"The endpoint of privileged runtime to communicate with")
 	enablePrivilegedRuntime = pflag.Bool("enable-privileged-runtime", true, "Enable privileged runtime to handle OS containers, default is true")
 	enableUnikernelRuntime  = pflag.Bool("enable-unikernel-runtime", false, "Enable unikernel runtime to run containers using unikernel image, default is false. Still under development.")
+	enableUnikernelLog      = pflag.Bool("enable-unikernel-log", true, "Enable unikernel runtime's log allow print VM's log to kubelet specified file, while disable some ability when `virsh console` to VM.")
```

Comment:
why needs an extra flag? I think it should be enabled by default.

Reply:
I'm not sure, if we enable log, there will be no console anymore, then there is no easy way for debugging. I think we can keep this ability until we found other ways to replace console.

Reply:
won't console's output be in logs?

Reply:
@feiskyer Yep, I think they are, while sometimes console can be interactive, could somehow replace `attach` for debugging purpose.

Reply:
ok, let's keep this flag.